### ISSUE TITLE
Change ScanSettings?

### ISSUE THREAD

**Issue by EternityForest** [b3b/able#25]
It seems that this uses a deprecated startLeScan function, and a newer API allows for filtering, and things like low power background scans. Is this functionality on the road map for the future?

It would be very useful for key tracker beacons and the like.

**Comment by b3b** [comment]
Yes, I have not check how difficult it would be to implement background scans, but switch to BluetoothLeScanner methods is planned.
